js/main.js

```diff
@@ -154,3 +156,41 @@ $entriesPage.addEventListener('click', function () {
     }
   }
 })
+
+var $openModal = document.querySelector('.modal-background')
+var $deleteLink = document.querySelector(".delete")
+$deleteLink.addEventListener('click', deleteEntry);
+function deleteEntry(event){
+    event.preventDefault();
+    $openModal.className = "modal-background"
+}
+
+
+var $hideButton = document.querySelector('.grey-button')
+$hideButton.addEventListener('click', hideModal)
+function hideModal(event){
+  event.preventDefault();
+  $openModal.className = "modal-background hidden"
+}
+
+var $deleteButton = document.querySelector('.red-button');
+$deleteButton.addEventListener('click', removeEntry)
+
+function removeEntry(event){
+  event.preventDefault();
+     for (var w = 0; w < data.entries.length; w++) {
+       if (data.editing === data.entries[w]) {
+         data.entries.splice(w, 1)
+         var $findLi = document.querySelectorAll('li')
+         for (var p = 0; p < $findLi.length; p++) {
+           if (parseInt($findLi[p].attributes['data-entry-id'].value) === data.editing.entryId) {
+             $findLi[p].remove();
+             data.nextEntryId--
```

Comment:
You should remove this line. Just because an entry was deleted doesn't mean the next id should go down. That would create a possibility of multiple entries with the same id, which would cause bugs.